### PR TITLE
refactor: refine some msgs

### PR DESCRIPTION
**Pull request by hi-rustin**
close https://github.com/ti-community-infra/ti-community-bot/issues/15

### CODE REVIEW

**Comment by hi-rustin** [comment]
/hold

Integration testing is required.

**Comment by hi-rustin** [comment]
/unhold

Test it in https://github.com/ti-community-infra/test-dev/pull/391 and https://github.com/ti-community-infra/test-dev/pull/390

/assign @Mini256 

PTAL~ Thanks!

**Review by Mini256 — APPROVED** [review]

File: src/events/pull/index.ts [hunk 1]
```diff
@@ -75,8 +70,11 @@ async function checkPullFormat(context: Context, pullService: PullService) {
     sha: head.sha,
     state: reply.status === Status.Success ? "success" : "failure",
     target_url: "https://github.com/ti-community-infra/ti-community-bot",
-    description: reply.message,
-    context: "Sig Info File Format",
+    description:
+      reply.message.length > GITHUB_STATUS_DESC_MAX_LENGTH
+        ? reply.message.substr(0, 130) + "..."
```
Comment (Mini256) [inline src/events/pull/index.ts:75]: Maybe 130 is calculated by the `GITHUB_STATUS_DESC_MAX_LENGTH` constant will be better.

**Review by Mini256 — CHANGES_REQUESTED** [review]

File: src/events/pull/index.ts (hunk 1 above)
Comment (hi-rustin) [reply src/events/pull/index.ts]: Fixed.

**Review by Mini256 — APPROVED** [review]

**Comment by hi-rustin** [comment]
@Mini256 Thanks!

**Pull request merged**